app/common/lib/bookmarkFoldersUtil.js

```diff
@@ -117,11 +118,32 @@ const isMoveAllowed = (folderList, sourceDetail, destinationDetail) => {
   return true
 }
 
+const buildFolder = (folderDetails, folders) => {
+  folderDetails = makeImmutable(folderDetails)
+  let key = folderDetails.get('folderId')
+
+  if (!folderDetails.has('folderId')) {
+    key = getNextFolderId(folders)
+  }
+
+  return makeImmutable({
+    title: folderDetails.get('title'),
+    folderId: ~~key,
```

Comment:
nit: I know it was similar before this change but this double `not` in bitwise is hard to read. I don't think we need to floor numbers in here so my guess is that we use it only to coerce from string to number? if so a more readable way would be to explicitly coerce to number like you did in #10610.

Reply:
Changed `~~` to `Number`. Note that I didn't changed `parentId` conversion, because I already changed that in `Other bookmarks` PR